src/vs/workbench/contrib/chat/browser/chatSessions.contribution.ts

```diff
@@ -633,6 +637,7 @@ class CodingAgentChatImplementation extends Disposable implements IChatAgentImpl
 						request,
 						prompt: request.message,
 						history,
+						metadata
```

Comment:
I am deleting this dynamic agent with https://github.com/microsoft/vscode/pull/268089